release_notes/374-release-notes.adoc

```diff
@@ -0,0 +1,321 @@
+
+{product-title-short} version 3.74 is the last release that supports {ocp} versions 4.6, 4.7, 4.8, and 4.9. {product-title-short} version 4.0 will not support these {ocp} versions or earlier releases. See the link:https://access.redhat.com/node/5822721[Red Hat Advanced Cluster Security for Kubernetes Support Policy] for more information.
+
+[id="bug-fixes-374"]
+== Bug fixes
+
+[id="resolved-in-version-3740"]
+=== Resolved in version 3.74.0
+
+* The {product-title-short} Operator previously incorrectly removed owner references from the `stackrox-db` persistent volume claim (PVC). This can cause an error such as `Failed reconciling PVC "stackrox-db". Please remove the storageClassName and size properties from your spec [...]`. The bug is fixed. However, if you receive this message, you need to perform the steps described in the link:https://access.redhat.com/solutions/6995414[Knowledgebase article]. (ROX-14335)
+* The `v1/alerts/summary/counts` endpoint, which is used by the *Policy violations by severity* and *Policy violations by category* widgets on the Dashboard, was fixed to prevent Central from crashing in highly-scaled environments. (ROX-13829)
+* {product-title-short} previously was not able to correctly identify the signature of images that contained the same content but were stored in different registries, causing problems with alerts and enforcement. This issue has been fixed and {product-title-short} will now correctly identify signatures of images from different registries. (ROX-12400)
+
+[id="known-issues-374"]
+=== Known issues
+
+* Currently, {product-title-short} does not support alerts for security policy violations for containers running with default `seccomp profiles-Unconfined`. The alert violations for Unconfined `seccomp` profiles are generated only if the `seccomp` profile is explicitly set to "Unconfined" in the container specification. There is no workaround. (ROX-13490)
+* In the {product-title-short} portal, the *Platform Configuration* -> *Clusters* page does not display information in the *Cloud Provider* field for Azure Red Hat OpenShift and Red Hat OpenShift Service on AWS (ROSA) clusters. There is no workaround. (ROX-14399)
+
+
+[id="image-versions-373"]
+== Image versions
+
+[%header,cols="1,1,2"]
+|===
+| Image | Description | Current version
+
+| Main
+| Includes Central, Sensor, Admission controller, and Compliance. Also includes `roxctl` for use in continuous integration (CI) systems.
+a| `registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:3.74`
+
+| Scanner
+| Scans images and nodes.
+a|`registry.redhat.io/advanced-cluster-security/rhacs-scanner-rhel8:3.74`
+
+| Scanner DB
+| Stores image scan results and vulnerability definitions.
+a|`registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:3.74`
+
+| Collector
+| Collects runtime activity in Kubernetes or {ocp} clusters.
+a| * `registry.redhat.io/advanced-cluster-security/rhacs-collector-rhel8:3.74` +
```

Comment:
It doesn't seem to be hurting anything, but there is an extra "+" at the end of this line. It shouldn't be necessary though when having lists in tables